### PR TITLE
Feature/fma 69/UI improvements

### PR DESCRIPTION
**Pull request by zoemaas**

### CODE REVIEW

File: app.config.js [hunk 1]
```diff
@@ -11,8 +11,7 @@ module.exports = {
     process.env.GATSBY_METADATACACHE_URI || 'https://aquarius.delta-dao.com',
 
   complianceUri:
-    process.env.GATSBY_COMPLIANCE_URI ||
-    'https://cs.gx.test.sphereon.com/api/2210vp',
+    process.env.GATSBY_COMPLIANCE_URI || 'https://cs.gx.test.sphereon.com/api',
```
Comment (sksadjad) [inline app.config.js:14]: I'm missing something here.. what is the role of fma compliance here?

**Review by sksadjad — COMMENTED** [review]
looks ok. I just have one question for @nklomp where are we referencing fma compliance here?

**Review by sksadjad — APPROVED** [review]
LGTM

File: app.config.js (hunk 1 above)
Comment (zoemaas) [reply app.config.js:14]: I don't think this question is related to the changes in the code. So, let's move on with the review and check with Niels later?

**Pull request merged**